ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -0,0 +1,633 @@
+                  />
+                  <FormTextField
+                    label={
+                      <>
+                        {t('tokenSymbol')}
+                        {symbolAutoFilled && !forceEditSymbol && (
+                          <ButtonLink
+                            onClick={() => setForceEditSymbol(true)}
+                            textAlign={TextAlign.End}
+                            paddingInlineEnd={1}
+                            paddingInlineStart={1}
+                            color={TextColor.primaryDefault}
+                          >
+                            {t('edit')}
+                          </ButtonLink>
+                        )}
+                      </>
+                    }
+                    value={customSymbol}
+                    onChange={(e) => handleCustomSymbolChange(e.target.value)}
+                    helpText={customSymbolError}
+                    error={customSymbolError}
+                    disabled={symbolAutoFilled && !forceEditSymbol}
+                    marginTop={6}
+                    inputProps={{
+                      'data-testid': 'import-tokens-modal-custom-symbol',
+                    }}
+                  />
+                  <FormTextField
+                    label={t('decimal')}
+                    type="number"
+                    value={customDecimals}
+                    onChange={(e) => handleCustomDecimalsChange(e.target.value)}
+                    helpText={customDecimalsError}
+                    error={customDecimalsError}
+                    disabled={decimalAutoFilled}
+                    min={MIN_DECIMAL_VALUE}
+                    max={MAX_DECIMAL_VALUE}
+                    marginTop={6}
+                    inputProps={{
+                      'data-testid': 'import-tokens-modal-custom-decimals',
+                    }}
+                  />
+                  {customDecimals === '' && (
+                    <BannerAlert severity={Severity.Warning}>
+                      <Text fontWeight={FontWeight.Bold}>
+                        {t('tokenDecimalFetchFailed')}
+                      </Text>
+                      {t('verifyThisTokenDecimalOn', [
+                        <ButtonLink
+                          key="import-token-verify-token-decimal"
+                          rel="noopener noreferrer"
+                          target="_blank"
+                          href={blockExplorerTokenLink}
+                        >
+                          {blockExplorerLabel}
+                        </ButtonLink>,
+                      ])}
+                    </BannerAlert>
+                  )}
+                </Box>
+              </Tab>
+            </Tabs>
+            <Box padding={6}>
+              <ButtonPrimary
+                onClick={() => handleNext()}
+                disabled={Boolean(hasError()) || !hasSelected()}
+                block
+              >
+                {t('next')}
+              </ButtonPrimary>
```

Comment:
Button size should be LG for modals right @SaraCheikh ?
```suggestion
              <ButtonPrimary
                size={Size.LG}
                onClick={() => handleNext()}
                disabled={Boolean(hasError()) || !hasSelected()}
                block
              >
                {t('next')}
              </ButtonPrimary>
```